hack/code/prices_gen.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws/session"
+	ec22 "github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/karpenter/pkg/cloudprovider/aws"
+	"go/format"
+	"io/ioutil"
+	"log"
+	"os"
+	"runtime"
+	"runtime/pprof"
+	"sort"
+	"strings"
+	"time"
+)
+
+func main() {
+	flag.Parse()
+	if flag.NArg() != 1 {
+		log.Printf("Usage: %s -- pkg/cloudprovider/aws/zz_generated.pricing.go", os.Args[0])
```

Comment:
why `--` here?

Reply:
Otherwise `go run` thinks that the generated file is part of the compilation package.

Reply:
oooooo because the file ends in `.go` I guess?  Was trying to compare against the docgen args

Reply:
You could always use something like https://github.com/spf13/cobra to parse the args. Totally optional, since I kinda hate cobra.

Reply:
It's actually not needed in that usage line, only used with `go run`. I updated the comment.